docs/sdk/src/guides/your_first_node.rs

```diff
@@ -245,7 +245,7 @@ mod tests {
 			.arg("--tmp")
 			.args(["--chain", chain_spec_file.to_str().unwrap()])
 			.args(["--dev-block-time", block_time.to_string().as_str()])
-			.timeout(std::time::Duration::from_secs(10))
+			.timeout(std::time::Duration::from_secs(30))
```

Comment:
Were tests here falling? A few of them are ignored, being flaky. Other than this, I was looking into spawning the node here with `.spawn()` instead of `.output()`, and redirect stdout/stderr, so that we can process them while the process is running. This will allow us to wait exactly how long is needed until a certain number of blocks are imported. We can also scan against errors and panic when found. Increasing the timeout works too, but we don't need to run the tests for 30 seconds most of the times.